utils/firestore.js

```diff
@@ -1237,6 +1237,100 @@ const searchSpecimen = async (masterSpecimenId, siteCode, allSitesFlag) => {
     return {};
 }
 
+/**
+ * getSiteLocationBox returns array of a single box object that matches the site and boxId
+ * @param {number} requestedSite - site code of the site
+ * @param {string} boxId - boxId of the box
+*/
+const getSiteLocationBox = async (requestedSite, boxId) => {
+    try {
+        const snapshot = await db.collection('boxes')
+                                .where(fieldMapping.loginSite.toString(), "==", requestedSite)
+                                .where(fieldMapping.shippingBoxId.toString(), "==", boxId).get();
+        const boxMatch = [];
+        for(let document of snapshot.docs) {
+            boxMatch.push(document.data());
+        }
+        return boxMatch;
+    } catch (error) {
+        throw new Error(`getSiteLocationBox() error: ${error.message}`);
+    }
+}
+
+/**
+ * getBiospecimenCollectionIdsFromBox returns an array of collectionIds from the box
+ * @param {number} requestedSite - site code of the site
+ * @param {string} boxId - boxId of the box
+*/
+const getBiospecimenCollectionIdsFromBox = async (requestedSite, boxId) => {
+    try {
+        const shipBoxMatch = await getSiteLocationBox(requestedSite, boxId);
+
+        if (shipBoxMatch === undefined || shipBoxMatch.length === 0) return [];
+        else {
+            let shipBoxObj = shipBoxMatch[0];
```

Comment:
Do we change value of `shipBoxObj`. If not should we use const instead of let

Reply:
That's a good point. Thanks for catching my incorrect variable declaration. I'll change it to `const` since there is no reassignment.